### ISSUE TITLE
Could you share Training Log?

### ISSUE THREAD

**Issue by joongkyunKim** [taylover-pei/SSDG-CVPR2020#6]
Hi, i'm trying to train SSDG to another dataset. 
I trained Oulu+SiW+ReplayAttack(which i could acquire) and tested another dataset.
The best results are achieved in just 2 epochs.. and i think it is a little bit odd..
So i want to check your training log. Could you share?

![image](https://user-images.githubusercontent.com/11433099/88000766-7a151d80-cb39-11ea-8fde-231b30388de9.png)

**Comment by taylover-pei** [comment]
Sorry for answering late.

Unfortunately, the original file is gone for some reason. But I have some suggestions for you:

1. There exist large differences between face anti-spoofing databases, so changing another two databases for training and testing needs some modification on the parameters.

2. The best results are achieved in just 2 epochs. It is definitely a bad result and cannot be reported. I haven't met this problem. I think you can reduce the learning rate or change the optimizer for some trying.

Good luck.
